client/src/components/Registration.js

```diff
@@ -81,22 +87,27 @@ class RegForm extends React.Component {
   }
 
   createUser(dataObject) {
-    var self = this;
+    let self = this;
     fetch('/api', {
       method: 'post',
       body: JSON.stringify(dataObject),
       headers: myHeaders
     })
     .then(function (response) {
```

Comment:
why not to use arrow functions to avoid usage of self?

Reply:
Yes, i have already done it)